docs/pandas_supported.rst

```diff
@@ -675,7 +675,7 @@ Performance will be affected by this.
 +--------------------+--------------------+----------------------------------------------------+
 | ``read_pickle``    | Y                  | Defaults to pandas implementation                  |
 +--------------------+--------------------+----------------------------------------------------+
-| ``read_sql``       | Y                  | Defaults to pandas implementation                  |
+| ``read_sql``       | Y                  | Distributed if used with extra arguments           |
```

Comment:
This is good for now, but we'll eventually want an Advanced Users page that this links to to tell users how to use it.